tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/TlaFunOper.scala

```diff
@@ -103,27 +105,29 @@ object TlaFunOper {
     *
     * <p>
     * `TlaOperDecl("Fact", List(),
-    *   OperEx(recFunDef,
```

Comment:
@konnov do want to include these formatting changes to the doc comments, or was this inadvertent?

Reply:
Let's skip this change. This must have been automatically reformatted